tests/behat/features/capabilities/event/filter-community-events.feature

```diff
@@ -0,0 +1,50 @@
+@api @javascript @event @stability @stability-1 @filter-community-events
+Feature: Filter community events
+  Benefit: Ability to filter events on All events page
+  Role: As a Verified
+  Goal/desire: I want to filter events that have all day checkbox activated
+
+  Scenario: Successfully use filters for ongoing and upcoming events with today's end date
+    Given users:
+      | name               | mail             | status | timezone | roles    |
+      | regular_user       | some@example.com | 1      | UTC      | verified |
+    Given events with non-anonymous author:
+      | title                     | body        | field_content_visibility | field_event_date    | field_event_date_end | field_event_all_day	 | langcode |
+      | Upcoming or Ongoing Event | lorem ipsum | public                   | today               | today                | 1                    | en       |
+    When I am logged in as "regular_user"
+    Given I am on the event overview
+    And I should see "Filter" in the "Sidebar second"
+    And I should see "Ongoing and upcoming events"
+    And I click radio button "Ongoing and upcoming events"
+    And I press "Filter"
```

Comment:
Here is our act block where we're doing things :) I'm adding a blank line before the block to ease readability and using the `When` keyword to signal acting.

I've removed the `I should see` assertions since we have the `Filter` assertion in other tests, and the clicking of the radio button will already make sure the test exists (so the duplicate `I should see "Ongoing and upcoming events"` isn't needed).
```suggestion

    When I am on the event overview
    And I click radio button "Ongoing and upcoming events"
    And I press "Filter"
```